src/tools/elektrad/kdb_handler.go

```diff
@@ -157,15 +166,93 @@ func (s *server) putKdbHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// putAllKdbHandler creating new keys or updating the values of existing keys
+//
+// Arguments:
+//
+//	parentKey The key in the path.
+//	configurations a list of key-value pairs.
+//
+// Response Code:
+//
+//	201 Created if a new key was created.
+//	400 Bad Request if the key name is invalid or the body is not a JSON
+//
+// All keys will be created below the parent key.
+//
+// Example: `curl -X PUT -d '"world"' localhost:33333/kdb/user:/tests`
+func (s *server) putAllKdbHandler(w http.ResponseWriter, r *http.Request) {
+	var configurations keyConfigurationSet
+
+	parentKey := parseKeyNameFromURL(r)
+
+	if err := parseFor(r, &configurations); err != nil {
+		writeError(w, err)
+		return
+	}
+
+	errKey, err := elektra.NewKey(parentKey)
+
+	if err != nil {
+		internalServerError(w)
+		return
+	}
+
+	defer errKey.Close()
+
+	handle, ks := getHandle(r)
+
+	_, err = handle.Get(ks, errKey)
+
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+
+	if err := createOrResetValue(parentKey, configurations, ks); err != nil {
+		writeError(w, err)
+		return
+	}
+
+	created(w)
+}
+
+func createOrResetValue(parentKeyName string, configSet keyConfigurationSet, ks elektra.KeySet) error {
```

Comment:
```suggestion
func createOrUpdateValue(parentKeyName string, configSet keyConfigurationSet, ks elektra.KeySet) error {
```
since it also updates